lib/ssl/test/openssl_session_ticket_SUITE.erl

```diff
@@ -133,9 +136,10 @@ openssl_server_basic(Config) when is_list(Config) ->
 
     %% Configure session tickets
     ClientOpts = [{session_tickets, auto},
-                  {versions, ['tlsv1.2','tlsv1.3']}|ClientOpts0],
+                  {versions, ['tlsv1.2','tlsv1.3']}|
```

Comment:
you're replacing value in proplists so often ...
maybe worth having some `ssl_test_lib:replace/2` or `ssl_test_lib:set/2` helper?
I guess we never want a duplicate in Config ... maybe should be used by default?

Reply:
Yeah maybe a later re-factorization of the test suites are needed.